src/components/VRating/VRating.ts

```diff
@@ -0,0 +1,202 @@
+// Styles
+import '../../stylus/components/_rating.styl'
+
+// Components
+import VIcon from '../VIcon'
+
+// Mixins
+import Colorable from '../../mixins/colorable'
+import Sizeable from '../../mixins/sizeable'
+import Rippleable from '../../mixins/rippleable'
+import Themeable from '../../mixins/themeable'
+
+// Utilities
+import { createRange } from '../../util/helpers'
+import mixins from '../../util/mixins'
+
+// Types
+import { VNode, VNodeDirective, VNodeChildrenArrayContents } from 'vue'
+
+type ItemSlotProps = {
+  index: number
+  value: number
+  isFilled: boolean
+  isHalfFilled: boolean
+  isHovered: boolean
+  isHalfHovered: boolean
+  click: Function
+}
+
+/* @vue/component */
+export default mixins(
+  Colorable,
+  Rippleable,
+  Sizeable,
+  Themeable
+).extend({
+  name: 'v-rating',
+
+  props: {
+    backgroundColor: {
+      type: String,
+      default: 'secondary'
+    },
+    color: {
+      type: String,
+      default: 'primary'
+    },
+    emptyIcon: {
+      type: String,
+      default: '$vuetify.icons.ratingEmpty'
+    },
+    fullIcon: {
+      type: String,
+      default: '$vuetify.icons.ratingFull'
+    },
+    halfIcon: {
+      type: String,
+      default: '$vuetify.icons.ratingHalf'
+    },
+    halfIncrements: Boolean,
+    length: {
+      type: Number,
+      default: 5
+    },
+    readonly: Boolean,
+    showHover: Boolean,
```

Comment:
This should be `true` by default (or renamed to `noHover`)
Is there any reason at all to not show hover (except being readonly which is covered by `readonly` prop)?

Reply:
It's a stylistic choice, to show rating when mousing over or not. Personally I think it should be false by default, but perhaps we should have a vote?

Reply:
For now I'm okay with sticking to hover being an opted into prop.